tests/integration/test_backup_restore_new/test.py

```diff
@@ -1206,13 +1206,9 @@ def test_backup_all(exclude_system_log_tables):
 
     exclude_from_backup = []
     if exclude_system_log_tables:
-        system_log_tables = (
-            instance.query(
-                "SELECT concat('system.', table) FROM system.tables WHERE (database = 'system') AND (table LIKE '%_log')"
-            )
-            .rstrip("\n")
-            .split("\n")
-        )
```

Comment:
I need to convert a TSV result from ClickHouse to a list of lines in Python.
But it appears Python calculates `''.split("\n")` as `['']` (a list with one empty string) somehow, not an empty list. Which is not what I want. `splitlines()` suits this case better.